### PR TITLE
TG-49-50: Shared Inputs

### PR DESCRIPTION
**Pull request by pcorreadev**

### CODE REVIEW

File: src/Components/Shared/Input/InputSelect/Index.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      id={id}
+      name={name}
+      onChange={onChange}
+      value={value}
+      required={required}
+    >
+      {arrayToMap.map((item) => {
+        return (
+          <option selected={Boolean(item._id === itemValue)} key={item._id} value={item._id}>
+            {`${item.params}`}
```
Comment (samueltrilloRR) [inline src/Components/Shared/Input/InputSelect/Index.js:26]: Should return the "label" value

File: src/Components/Shared/Input/InputSelect/Index.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      className={className}
+      id={id}
+      name={name}
+      onChange={onChange}
+      value={value}
+      required={required}
+    >
+      {arrayToMap.map((item) => {
+        return (
+          <option selected={Boolean(item._id === itemValue)} key={item._id} value={item._id}>
```
Comment (samueltrilloRR) [inline src/Components/Shared/Input/InputSelect/Index.js:25]: "_id" prop does not exist in the objects that are defined inside "arrayToMap" array

File: src/Components/Shared/Input/InputSelect/select.module.css [hunk 3]
```diff
@@ -0,0 +1,9 @@
+select {
```
Comment (samueltrilloRR) [inline src/Components/Shared/Input/InputSelect/select.module.css:1]: Replace this "styled tag" with a class

File: src/Components/Shared/Input/Input.module.css [hunk 4]
```diff
@@ -0,0 +1,9 @@
+input {
```
Comment (samueltrilloRR) [inline src/Components/Shared/Input/Input.module.css:1]: Replace this "styled tag" with a class. Also, this stylesheet should be rename to "inputText.module.css" to change the initial letter (should no start with capital letter) and the file should be placed inside new folder called InputText

File: src/Components/Shared/Input/index.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import styles from './Input.module.css';
```
Comment (samueltrilloRR) [inline src/Components/Shared/Input/index.js:1]: Should be placed inside new folder called "InputText" to keep the consistency

File: src/Components/Shared/Input/index.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+const Input = (name, type, placeholder, value, onChange, required, className = styles) => {
```
Comment (samueltrilloRR) [inline src/Components/Shared/Input/index.js:3]: It should receive a "label" prop to show it and link it with the created input

File: src/Components/Shared/Input/index.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+  return (
+    <input
```
Comment (samueltrilloRR) [inline src/Components/Shared/Input/index.js:5]: Should be an "label" tag linked with the "input" tag

**Review by samueltrilloRR — CHANGES_REQUESTED** [review]

**Review by samueltrilloRR — CHANGES_REQUESTED** [review]

File: src/Components/Shared/Input/InputSelect/Index.js [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+        id={id}
+        name={name}
+        onChange={onChange}
+        value={value}
+        required={required}
+      >
+        {arrayToMap.map((item) => {
+          return (
+            <option selected={Boolean(item.id === itemValue)} key={item.id} value={item.id}>
+              {`${item.params}`}
```
Comment (samueltrilloRR) [inline src/Components/Shared/Input/InputSelect/Index.js:29]: Params should be renamed into "label"

File: src/Components/TimeSheets/Form/index.js [hunk 9]
```diff
@@ -79,6 +81,21 @@ const Form = () => {
...
+    return {
+      id: item._id,
+      params: `${item.firstName + ' ' + item.lastName}`
+    };
+  });
+
+  const arrayToMapProjects = projects.map((item) => {
+    return {
+      id: item._id,
+      params: `${item.name}`
```
Comment (samueltrilloRR) [inline src/Components/TimeSheets/Form/index.js:95]: "params" should be rename to a properly name property as "label"

**Review by samueltrilloRR — CHANGES_REQUESTED** [review]

File: src/Components/Shared/Input/InputText/inputText.module.css [hunk 10]
```diff
@@ -2,30 +2,21 @@
...
 .input {
-  display: flex;
-  justify-content: center;
   margin: auto;
-  align-items: center;
-  text-align: center;
-  width: 100%;
   box-sizing: border-box;
+  width: 100%;
 }
```
Comment (OctavioRodriguezPaseyro) [inline src/Components/Shared/Input/InputText/inputText.module.css]: no newline at end of inputText.module.css

File: src/Components/Shared/Input/InputSelect/inputSelect.module.css [hunk 11]
```diff
@@ -2,29 +2,20 @@
...
   }
-
   .select {
-    display: flex;
     width: 100%;
-    justify-content: center;
     margin: auto;
-    align-items: center;
-    text-align: center;
   }
```
Comment (OctavioRodriguezPaseyro) [inline src/Components/Shared/Input/InputSelect/inputSelect.module.css]: no newline at end of inputSelect.module.css

**Review by OctavioRodriguezPaseyro — APPROVED** [review]

**Review by samueltrilloRR — APPROVED** [review]

**Pull request merged**